GPTBackend/app/Http/Controllers/Controllers/Api/v1/RoomsController.php

```diff
@@ -3,18 +3,40 @@
 namespace App\Http\Controllers\Controllers\Api\v1;
 
 use App\Http\Controllers\Controller;
+use App\Http\Requests\FindRoomRequest;
 use App\Service\RoomsService;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
 
 class RoomsController extends Controller
 {
+    private $roomsService;
+    public function __construct()
+    {
+        $this->roomsService = new RoomsService();
+    }
+
     public function index()
     {
         try {
-            $rooms = new RoomsService();
-            return response()->json(['data' => $rooms->getList()], 200);
+            $rooms = $this->roomsService->getList();
+            return response()->json(['message' => 'Success',
+                                    'data' => $rooms], 200);
+        }catch(\Throwable $th){
+            return response()->json(['errors' => ['message' => $th->getMessage()]], 500);
+        }
+    }
+
+    public function findRoom()
+    {
+        try {
+            $request = request();
+            $rs = $this->roomsService->findRoom($request);
```

Comment:
@padi-dev-anhtt 
Lúc đặt tên thì không đươc viết tắt và chọn tên rõ nghĩa, tránh chọn tên chung chung kiểu `result`, `value`, `item`,...

```suggestion
            $rooms = $this->roomsService->findRoom($request);
```